src/Microsoft.AspNetCore.Mvc/project.json

```diff
@@ -11,7 +11,8 @@
     "nowarn": [
       "CS1591"
     ],
-    "xmlDoc": true
+    "xmlDoc": true,
+    "preserveCompilationContext": true
```

Comment:
We have a test that verifies if a list of package names we have is near identical to this assembly's dependencies. In the past we used the `ILibraryManager` to discover these dependencies.